<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skeletonization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Renderer" name="widgetMesh" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave_2"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Skeleton"/>
    <addaction name="actionIterateOnce"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load Mesh...</string>
   </property>
   <property name="iconText">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSaveMesh">
   <property name="text">
    <string>Export Mesh</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save Skeleton...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Skeletonization...</string>
   </property>
  </action>
  <action name="actionCreate_Skeleton">
   <property name="text">
    <string>Create Skeleton</string>
   </property>
  </action>
  <action name="actionIterateOnce">
   <property name="text">
    <string>Itereate Once</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Renderer</class>
   <extends>QWidget</extends>
   <header location="global">Renderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
